torch/_dynamo/variables/higher_order_ops.py

```diff
@@ -845,4 +934,31 @@ def call_function(
 
 
 class CheckpointHigherOrderVariable(WrapHigherOrderVariable):
-    pass
+    def call_function(
```

Comment:
Had to bring this change from https://github.com/pytorch/pytorch/pull/104577 to get the following test passing

https://github.com/pytorch/pytorch/blob/17ab4f85e950761eba081f0874abdb782a012fcf/test/dynamo/test_higher_order_ops.py#L1943